test/services/ConsignmentStatusServiceSpec.scala

```diff
@@ -0,0 +1,52 @@
+package services
+
+import com.nimbusds.oauth2.sdk.token.BearerAccessToken
+import configuration.GraphQLConfiguration
+import graphql.codegen.GetConsignmentStatus.getConsignmentStatus.GetConsignment.CurrentStatus
+import org.mockito.Mockito
+import org.scalatest.{BeforeAndAfterEach, Matchers, WordSpec}
+import graphql.codegen.GetConsignmentStatus.{getConsignmentStatus => gcs}
+import org.mockito.Mockito.when
+import org.scalatestplus.mockito.MockitoSugar
+import uk.gov.nationalarchives.tdr.{GraphQLClient, GraphQlResponse}
+import configuration.GraphQLBackend._
+import org.scalatest.concurrent.ScalaFutures.convertScalaFuture
+
+import java.util.UUID
+import scala.concurrent.{ExecutionContext, Future}
+
+class ConsignmentStatusServiceSpec extends WordSpec with Matchers with MockitoSugar with BeforeAndAfterEach {
+  implicit val ec: ExecutionContext = ExecutionContext.global
+
+  private val graphQLConfig = mock[GraphQLConfiguration]
+  private val getConsignmentStatusClient = mock[GraphQLClient[gcs.Data, gcs.Variables]]
+  private val token = new BearerAccessToken("some-token")
+  private val consignmentId = UUID.fromString("e1ca3948-ee41-4e80-85e6-2123040c135d")
+  when(graphQLConfig.getClient[gcs.Data, gcs.Variables]()).thenReturn(getConsignmentStatusClient)
+
+  private val consignmentStatusService = new ConsignmentStatusService(graphQLConfig)
+
+  override def afterEach(): Unit = {
+    Mockito.reset(getConsignmentStatusClient)
+  }
+
+  "consignmentStatus" should {
```

Comment:
Maybe add the word "function" to make it clear what is being tested: `"consignmentStatus function" should ...`